sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiverClient.cs

```diff
@@ -94,7 +51,10 @@ public class ServiceBusReceiverClient : IAsyncDisposable
         /// </summary>
         public ReceiveMode ReceiveMode { get; private set; }
 
-        internal bool IsSessionReceiver { get; set; }
+        /// <summary>
+        ///
+        /// </summary>
+        public bool IsSessionReceiver { get; set; }
```

Comment:
Should this be `private set`?   Do we want callers to be able to change the flag after creation?

Reply:
Fixed